lib/commands/init.js

```diff
@@ -34,6 +37,27 @@ module.exports = Command.extend({
     }
   },
 
+  beforeRun: function(rawArgs) {
+    if (!rawArgs.length) {
+      return;
+    }
+    // merge in blueprint availableOptions
+    var blueprint;
+    try{
+      blueprint = this.lookupBlueprint(rawArgs[0]);
+      this.registerOptions(blueprint);
+    }
```

Comment:
I would prefer the above mentioned style guide.

I would love to enforce style with JSCS or similar sometime soon, that way we can keep this consistent.